### ISSUE TITLE
potential error

### ISSUE THREAD

**Issue by RustyyES** [MahmudulHasan11085/Vision-Eagle-Attention#1]
I might be wrong here but it seems like you used the training set for the test_loader and it was for validation which means that you are validating on the same identical set , and also you used test_loader method to assess test accuracy but you simply used the validation set from before which wasn't correct in the first place , but I might be wrong here so kindly assess this issue here

**Comment by MahmudulHasan11085** [comment]
> Thank you for raising the issue. There are some errors in the fashionmnist notebook. I will try to update it asap. Another thing here is that Validation set and testing set are not different here. To print the confusion matrix and other matrices, I just calculated testing set once again.

**Comment by MahmudulHasan11085** [comment]
> I might be wrong here but it seems like you used the training set for the test_loader and it was for validation which means that you are validating on the same identical set , and also you used test_loader method to assess test accuracy but you simply used the validation set from before which wasn't correct in the first place , but I might be wrong here so kindly assess this issue here

Issues have been updated in the notebook. It will be updated in the arxiv paper and paperswithcode soon.
